docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -401,6 +579,29 @@ User configuration for AGP Pruner
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.AGPPruner
 
+Performance Test
+^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Model
+     - Accucacy
+     - Parameters
+     - FLOPs
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 76.94%
+     - 
+     - 
```

Comment:
why the two numbers are missing?

Reply:
Because the result was not saved at that time, now it has been added.